tests/unit/pages/tutor-profile/TutorProfile.spec.jsx

```diff
@@ -76,12 +84,60 @@ describe('TutorProfile', () => {
 
     useAxios.mockImplementation(() => fakeData)
     renderWithProviders(<TutorProfile />, {
-      initialEntries: route,
+      initialEntries: tutorRoute,
       preloadedState: { appMain }
     })
 
     const name = screen.getByText(`${mockData.firstName} ${mockData.lastName}`)
 
     expect(name).toBeInTheDocument()
   })
+
+  it('Should render video presentation block for tutor', () => {
+    const fakeData = getFakeData(false)
+
+    useAxios.mockImplementation(() => fakeData)
+    renderWithProviders(<TutorProfile />, {
+      initialEntries: tutorRoute,
+      preloadedState: { appMain }
+    })
+
+    const videoBlockTitle = screen.getByText(
+      'tutorProfilePage.videoPresentation.title'
+    )
+
+    expect(videoBlockTitle).toBeInTheDocument()
+  })
+
+  it('Should not render video presentation block when student has no video link', () => {
+    const fakeData = getFakeData(false)
+
+    useAxios.mockImplementation(() => fakeData)
+    renderWithProviders(<TutorProfile />, {
+      initialEntries: studentRoute,
+      preloadedState: { appMain }
+    })
+
+    const videoBlockTitle = screen.queryByText(
+      'tutorProfilePage.videoPresentation.title'
+    )
+
+    expect(videoBlockTitle).not.toBeInTheDocument()
+  })
+
+  it('Should render video presentation block when student has a link', () => {
+    const fakeData = getFakeData(false, videoMockData)
+
+    useAxios.mockImplementation(() => fakeData)
+    renderWithProviders(<TutorProfile />, {
```

Comment:
As I noticed, in each test we need to do the same setup, but with different parameters. I guess you can move this common logic to a separate function:
```javascript
const renderWithMockData = (load, initialEntries, extraData = {}) => {
  //mock and rendering logic
}
```
And then just reuse it:
```javascript
it('Should render video presentation block when student has a link', () => {
  renderWithMockData(false, studentRoute, videoMockData)
  ...
})
```

Reply:
Thanks. I changed it as you suggested in a new commit.